### ISSUE TITLE
`prefer-importing-vitest-globals` does not flag or fix imports that are immediately called with a method

### ISSUE THREAD

**Issue by stern-shawn** [vitest-dev/eslint-plugin-vitest#766]
First off, thanks a bunch for this plugin!

While using this to migrate away from `globals: true` to explicit imports, I noticed that the plugin with `prefer-importing-vitest-globals` enabled autofixes all direct instances where a `vitest` import should take place. For example, `it('...')`, `describe('...')`, etc all get auto-fixed and imported, great!

Unfortunately, instances like the following do not seem to be picked up by the linter and require manual fixes (which is a lot for a large mono-repo).

```ts
vi.spyOn();
it.each()
describe.skip();
```

The common difference between what works and what doesn't appears to be that the uncaught imports all deal with using a global import and immediately calling a method on it.

Have I possibly misconfigured something or is this working as intended? Thanks!
